templates/templates/preview/base_html.template

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<body>
+{% include "preview/experiment_html.template" %}
+{%- if manifest.all_entity_lists|length > 0 %}
+<p>=== Entity Preview ===<p/>
```

Comment:
- Consider using container elements to group items.
- Consider setting classes on semantic elements to enable applying CSS 
- Consider looping in the parent (base_html.template) for databases so the database template doesn't have to worry about "sets of databases"

```
<div class='preview'>   <------- container for a preview will enable things like expand/contract on the entire element  -->
    <div class='header'>Experiment Preview</div>   <------- header class lets us bold/font-face/font-size w/CSS
    <div class='experiment'>
        {% include "preview/experiment_html.template %}
    </div>
    <div class='childContainer'>     <------- could help us do things like alternate colors on rows for different types of children
    <div class='childEntity orchestrator'>  <----- can help us do clever things like click a filter in a front end to show only orchestrators, etc
        {% for orc in manifest.dbs %} <----- loop in parent
            {% include "preview/orchestrator_html.template" %}
        {% endfor %}
    </div>
    </div>
</div>
```